CHANGELOG.md

```diff
@@ -1,9 +1,23 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
-[Unreleased]: git+https://github.com/MetaMask/ens-resolver-snap/
+## [0.1.0]
+
+### Added
+
+- feat: implement ENS resolver snap (based on snap monorepo
+  template) ([#1](https://github.com/MetaMask/ens-resolver-snap/pull/1))
+
+### Changed
+
+- refactor: transform repository layout to single library instead of
+  monorepo ([#16](https://github.com/MetaMask/ens-resolver-snap/pull/16))
```

Comment:
This change doesn't affect any consumers, so doesn't need to be mentioned in the changelog.

```suggestion
```